src/VersionsCheck.php

```diff
@@ -101,7 +101,11 @@ public function versionCompare($version1, $operator, $version2)
 
     private function createNotUpToDateOutput(array &$output)
     {
-        $output[] = '<warning>Some packages are not up to date:</warning>';
+        $output[] = sprintf(
+            '<warning>%d %s not up to date:</warning>',
+            count($this->outdatedPackages),
+            1 != count($this->outdatedPackages) ? 'packages are' : 'package is'
```

Comment:
Using `count` twice will cost useless resources IMO.

Would be better to store the count on a result variable before like `$outdatedPackagesCount`.